Libraries/WebRTC/nvencodernobuffer.patch

```diff
@@ -0,0 +1,114 @@
+diff --git a/webrtc/modules/video_coding/codecs/h264/h264_encoder_impl.cc b/webrtc/modules/video_coding/codecs/h264/h264_encoder_impl.cc
+index 5705428a1..9b1710122 100644
+--- a/webrtc/modules/video_coding/codecs/h264/h264_encoder_impl.cc
```

Comment:
![](https://media2.giphy.com/media/14ut8PhnIwzros/giphy.gif)

^^ my reaction everytime i remember we do this with a patch. The change looks fine though, this gif is purely for entertainment purposes.

Reply:
Don't worry, I've done lots of testing for this change and it does bring better latency.